services/121-service/test/registrations/pagination/filter-registration.test.ts

```diff
@@ -166,6 +166,28 @@ describe('Load PA table', () => {
       expect(meta.totalItems).toBe(1);
     });
 
+    it('should filter with using a referenceId string that could be interpreted as number', async () => {
+      // Arrange
+      const referenceIdInterpretableAsNumber = '651581942751358e5'; // A just-number string like '1234567890' is somehow not sufficiently covering the edge case, as this would not lead to the same bug prior to bugfix AB#30713
```

Comment:
make this a shared variable between all these tests? maybe split these tests off in separate file, now that they are 5?

Reply:
Good call. I've split the tests into a separate file, and also reverted the other change that you mention in your other comment.